app/models/patient_import_row.rb

```diff
@@ -53,6 +53,18 @@ def to_patient
         existing_patient.gender_code = attributes.delete(:gender_code)
       end
 
+      if attributes[:preferred_given_name].present? &&
```

Comment:
I'm wondering if it's time to extract this common logic in to methods, where the attribute name can be passed in. The current method is getting quite long and has got a fair amount of duplication.